lms/static/js/edxnotes/views/tabs/tags.js

```diff
@@ -0,0 +1,117 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'gettext', 'jquery', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
+    'js/edxnotes/views/tab_view'
+], function (gettext, $, _, NoteGroupView, TabPanelView, TabView) {
+
+    var TagsView = TabView.extend({
+        scrollToTag: function(tagName) {
+            var titleElement;
+            if (!this.tabModel.isActive()) {
+                this.tabModel.activate();
+            }
+            titleElement = this.$el.find('.tags-title').filter(
+                // Since tags can only be single words, we are safe just looking for the title
```

Comment:
Note to reviewers: I'm going to have to make this algorithm smarter, as it won't handle the case of a tag being a substring of another tag (scroll to "cat" where the tag "catfood" also exists). I'll work on that today.